client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts

```diff
@@ -88,9 +89,10 @@ export class MediafileListComponent extends ListViewBaseComponent<ViewMediafile,
     public constructor(
         titleService: Title,
         matSnackBar: MatSnackBar,
+        storage: StorageService,
         protected translate: TranslateService,
         private router: Router,
-        private route: ActivatedRoute,
+        protected route: ActivatedRoute,
```

Comment:
no modifier and move this up, so the order of arguments mathces with the `super` call.